app/views/static/index.html.erb

```diff
@@ -1,92 +1,76 @@
-<figure class="beer-list-logo-container">
-  <%= image_tag('beer-list.svg', class:"beer-list-logo") %>
-</figure>
+<%= image_tag ("beer-list.svg"), :id => "beer-list" %>
+<div id="white-container">
+  <div id="beers">
 
-<div class="white-container">
-  <% @beers.each_slice(2) do |work| %>
-  <div class="columns beers">
-    <% work.each do |beer| %>
-    <div class="column is-half beer">
-      <div class="columns is-gapless">
-        <div class="column">
-          <figure class="image square beer-logo-crop">
-            <% if beer.image.attached? %>
-            <%= image_tag(beer.image, class:"beer-logo") %>
-            <% else %>
-            <%= image_tag('default-beer-image.svg', class: 'beer-logo') %>
-            <% end %>
-          </figure>
-        </div>
-        <div class="column is-three-fifths beer-text">
-          <div class="beer-name"><%= beer.name%></div>
-          <div class="beer-tagline"><%= beer.tag_line%></div>
-        </div>
-        <div class="column beer-tier">
-          <div class="beer-tier-text">TIER</div>
-          <div class="beer-tier-content"><%= beer.tier.short_hand %></div>
-        </div>
-        <div class="column beer-abv">
-          <div class="abv-text">ABV</div>
-          <div class="abv-content"><%= beer.abv%>%</div>
-        </div>
-      </div>
-    </div>
-    <% end %>
-  </div>
+  <% @beers.each do |beer| %>
+		<div class= "beer">
+			<div class="logo-and-text">
+				<% if beer.image.attached? %>
+				<%= image_tag(beer.image, class: 'beer-logo') %>
+				<% else %>
+				<%= image_tag('default-beer-image.svg', class: 'beer-logo') %>
+				<% end %>
+				<div class="beer-text">
+					<div class="beer-name"><%= beer.name %></div>
+					<div class="beer-tagline"><%= beer.tag_line %></div>
+				</div>
+			</div>
+		        <div class="abv-and-tier">
+		        	<div class="beer-abv">
+		          		<div class="abv-text">ABV</div>
+				        <div class="abv-content"><%= beer.abv %>%</div>
+		        	</div>
+		       		<div class="beer-tier">
+		         		<div class="beer-tier-text">TIER</div>
+		          		<div class="beer-tier-content"><%= beer.tier.short_hand %></div>
+		        	</div>
+		      	</div>
+		</div>
   <% end %>
 
-  <div class="guest-beers">
-    <% unless @guest_beers.empty? %>
-    <div class="columns">
-      <div class="column">
-        <%= image_tag('wine-cider.svg') %>    
-      </div>
-      <% @guest_beers.each do |guest_beer|%>
-      <div class="column">
-        <div class="columns is-gapless">
-          <div class="column has-text-centered">
-            <%= guest_beer.name %>
-            <span class="has-text-grey-light">/</span>
-            $<%= guest_beer.price %>
+
+    <!-- Wine & Cider List -->
```

Comment:
This version of the application does have a data model for these items, though they are not exposed in ActiveAdmin yet. Not really sure if we should opt to include them in the view and ditch the "Guest Beers" object (already a misnomer).